src/main/java/step2/Calculator.java

```diff
@@ -0,0 +1,75 @@
+package step2;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import static step2.Operator.PLUS;
+
+public class Calculator {
+
+    private int result = 0;
+
+    private Calculator() {
+
+    }
+
+    public Calculator(String input) {
+        final List<String> values = getInputsBySplitWhiteSpace(input);
+        final List<Integer> numbers = getNumbersByInputs(values);
+        final List<Operator> operators = getOperationsByInputs(values);
+
+        int index = 0;
+        for (Operator operator : operators) {
+            int prev = result;
+            int next = numbers.get(index++);
+            result = operator.operate(prev, next);
+        }
+    }
+
+    public static boolean isBlank(String value) {
+        return Objects.isNull(value) || "".equalsIgnoreCase(value);
+    }
```

Comment:
이 메서드만 static인 이유가 따로 있을까요??

Reply:
```suggestion
    private boolean isBlank(String value) {
        return Objects.isNull(value) || "".equalsIgnoreCase(value);
    }
```

에구 놓쳤네요!! 잡아주셔서 감사합니다.